GymApp/GymAppApi/Controllers/CouchesController.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace GYM.API.Controllers
 {
-    [Authorize]
+    [Authorize("AllMethodsAllowed")]
```

Comment:
When you use Authorize attribute you can delete thes same attributes from specific endpoints (like your GET endpoint in this controller)
Please, review the rest of endpoints as well